page.php

```diff
@@ -25,4 +25,8 @@
 
 $timber_post     = new Timber\Post();
 $context['post'] = $timber_post;
-Timber::render( array( 'page-' . $timber_post->post_name . '.twig', 'page.twig' ), $context );
+if (post_password_required($timber_post->ID)) {
+  Timber::render("single-password.twig", $context);
+} else {
+  Timber::render( array( 'page-' . $timber_post->post_name . '.twig', 'page.twig' ), $context );
+}
```

Comment:
```suggestion
Timber::render(
	post_password_required( $timber_post->ID )
		? 'single-password.twig'
		: array( 'page-' . $timber_post->post_name . '.twig', 'page.twig' ),
	$context
);
```

How about this?

Reply:
personally, I prefer the more verbose conditional vs. the ternary one... but maybe that's just me. 🤷‍♂️

Reply:
I like the big sack in a small sack, you like the small sack in the big one :)